Cython/Compiler/ExprNodes.py

```diff
@@ -11223,6 +11229,10 @@ def analyse_c_operation(self, env):
             self.operand2 = self.operand2.coerce_to(self.type, env)
 
     def compute_c_result_type(self, type1, type2):
+        if type1.is_cfunction and type1.is_cgetter:
+            type1 = type1.return_type
+        if type2.is_cfunction and type2.is_cgetter:
+            type2 = type2.return_type
```

Comment:
I actually wonder if `is_cgetter` should be a feature of the type or of the entry. If the latter, then the code that calls this method should do the checking here. It doesn't really feel like a type specific thing to me, at least…

Reply:
moved `c_getter` to `AttributeNode`

Reply:
It didn't work. The `@property` decorator is set on the `CFuncDefNode`. The `compute_c_result_type` is called for the `AttributeNode`. They both share a common `entry` instance. How can I transfer the `is_cgetter` set in `ReplacePropertyNode.visit_CFuncDefNode` to the `AttributeNode`that shares the entry?

Reply:
Through the `Entry`. :) Symbol table entries are shared within their scope.
Add a class attribute `is_cgetter = False` to the `Entry` class and then set it to `True` on the instance where you need it.

Reply:
Confusing. In order to move the `is_getter` off the entry, I have to set it on the entry anyway? Wouldn't it be simpler to just leave it on the entry?